src/Console/CommandRunner.php

```diff
@@ -172,6 +172,22 @@ public function run(array $argv, ?ConsoleIo $io = null): int
             $result = $this->runCommand($shell, $argv, $io);
         }
 
+        $event = $this->dispatchEvent('Console.afterRun', [
```

Comment:
I don't think this the right approach. I would expect the event to fire after every command not after the first command run from the console.

Reply:
I especially didn't want to to trigger it on every command run because one could easily produce an infinite loop with that (unless we want to catch that)

Reply:
That's why I'm not sure an after run event is the right approach for executing another command.